lms/djangoapps/labster_course_license/utils.py

```diff
@@ -105,8 +106,13 @@ def course_tree_info(store, simulations, licensed_simulations):
     Retuns information about the course's xblocks.
     """
     course_info = {}
+    invalid_simulations = []
+    validation_pattern = r'^https:\/\/[\w\d]*\.labster\.com\/[a-z]*\/[\w\d]*\/$'
```

Comment:
in this case it doesn't work with local enviroment.
I'd suggest to use 2 validations:
1. Validate general url format.
2. Validate simulation id.